docs/tutorials/tutorial-jupyter-docker.md

```diff
@@ -85,6 +88,17 @@ In the same directory as `docker-compose.yaml` and `environment`, start the appl
 DRUID_VERSION={{DRUIDVERSION}} docker compose --profile all-services up -d
 ```
 
+### Start Kafka and Jupyter
```

Comment:
Would someone need or want to do this if they weren't on ARM? We should probably add some more context on what and how to use this.

IE this starts Jupyter and Kafka only with the expectation that they have an external Druid to use, either a cluster they've set up in the past or they want to run the quickstart Druid externally for whatever reason. And they'll need to update the host variable